sysutils/go-mmv/Portfile

```
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/itchyny/mmv 0.1.2 v
+name                go-mmv
+revision            0
+
+description         rename multiple files with editor
+
+long_description    Rename multiple files using your \$EDITOR. The command \
+                    name is named after multi-mv.
+
+conflicts           mmv
+
+categories          sysutils
+license             MIT
+platforms           darwin macosx
```

```suggestion
platforms           darwin
```

I hadn't seen `macosx` in a portfile before, so I looked it up in the docs (thanks for introducing it to me).

From https://guide.macports.org/#reference.keywords, `darwin` is "equivalent to specifying both macosx and puredarwin". So my understanding is that either one should be used, but not both.

---

Good catch @harens - fixed